CLASSES/class_person.py

```diff
@@ -0,0 +1,92 @@
+class Person:
+    max_health = 100
+    base_stats = ("attack", "defense", "intelligence", "health")
+
+    def __init__(self, features: dict):
```

Comment:
Fajnie, że przekazujesz słownik. O to chodziło żeby wszystko, co potrzeba trafiło do klasy. Trzeba tylko popracować nad tym jak on jest zbudowany.